packages/venia-concept/src/index.css

```diff
@@ -99,6 +99,7 @@ button {
     padding: 0;
     touch-action: manipulation;
     user-select: none;
+    outline: none;
```

Comment:
It is considered a bad practice to remove the outline from an active element (http://www.outlinenone.com/).

I would start by asking why the outline on the button feels wrong. 

Perhaps, the search button should move the focus to the input when the button is tapped? 
Or the style of the outline does not meet the expectations?

Reply:
@belbiy good remark, thank you. I read the article you provided and I agree with you that the outline should be left on the desktop, but do we need it on mobile devices?

Reply:
It also seems to me that we need to move the focus after we have clicked on the active element. **For example:** Click on the search element, and after, the focus is transferred to the input field to search.

Reply:
[Removing that ugly :focus ring (and keeping it too)](https://hackernoon.com/removing-that-ugly-focus-ring-and-keeping-it-too-6c8727fefcd2)

Reply:
@DmitryShmaliuk, we cant just remove the focus ring for mobile. There's an interesting prototype for [input modality](https://www.oreilly.com/ideas/proposing-css-input-modality) that handles the keyboard-mouse input switching.

But in reality, i think that we can improve user experience greatly by doing simple things first:

1. For search button, move the focus to the input field, so the user does not need to make extra tap. Consider the same behavior for every button – tapping a button can move focus the the area that button controls. If the button opens a shopping cart, for example, move focus inside that modal, etc. This can be a feature of the modal component actually, receiving focus on open, possibly even "trapping" the focus inside itself until closed.

2. Styling the `:focus` state of the buttons and other active elements that they look consistent and not "weird" when focused.

 @soumya-ashok, what do you think?

Reply:
@belbiy Agree with your feedback. I'd just be cautious with choosing the default focus area in a scenario like the cart where there are multiple focusable elements and there aren't traditional form fields in the same way search or checkout has. We should choose a default that causes the least friction for the user in this scenario.

Reply:
@belbiy, i agree with you, and i will try to implement the your points.